### ISSUE TITLE
Could really use an good NVDA Addin or the like for the MCP-D74 program

### ISSUE THREAD

**Issue by weekscasey** [nvaccess/nvda#8773]
I have downloaded the MCP-D74 program from kenwood's site and found that some things do not read well if at all. The main areas that are causing troubles seem to be treeviews and tables. This peace of software is useful for programming the kenwood TH-d74a.
 ### Steps to reproduce:
Bring up  the program mcp-74.
Then, get through the main setup screens shown when program is launched for the first time.
If tabbing around the main screen does read ok, then choose something like repeater lookup and or call sign search and or and anything in the edit menu of this program that even looks like it may contain a table and or treeview. NVDA will not read either all that well if at all.
I know that there was some JAWS scripts at the following URL, but I doudt that would be of much help seeing that this is NVDA here.
https://active-elements.org/2017/12/15/kenwood-th-d74-mcp-jaws-scripts

### Actual behavior:
Rows and colams numbers are about the only thing that is read when working in a area that is a table in this program.

### Expected behavior:
All buttens being read allowed. Contents of tables and any treeviews and the like should be available to the NVDA user of this program. Even when there is areas that can and or be changeable by said user. Errowing and or tabbing  around the screen should read all butten tables, treeview, and menu things aloud.
#### NVDA Installed/portable/running from source:
installed on hard drive

[nvda.log](https://github.com/nvaccess/nvda/files/2408198/nvda.log)
#### NVDA version:
2018.3.1

#### Windows version:
OS Name	Microsoft Windows 7 Professional

Version	6.1.7601 Service Pack 1 Build 7601
System Type	x64-based PC
Processor	Intel(R) Core(TM) i5-4590S CPU @ 3.00GHz, 3001 Mhz, 4 Core(s), 4 Logical Processor(s)
Installed Physical Memory (RAM)	16.0 GB

#### Name and version of other software in use when reproducing the issue:
mcp-74 v1.03
This software can be gotten from the following URL.
http://www.kenwood.com/i/products/info/amateur/mcpd74_e.html
#### Does the issue still occur after restarting your PC?
Yes it does

#### Have you tried any other versions of NVDA?

Yes, a version or two before the latest version

**Comment by Brian1Gaff** [comment]
Would one be able to test this software without a Kenwood Ham radio attached 
though? I'm assuming this is what the software is for. Icom have similar 
software  but I am not a ham nor have I tried it.
 In the states I believe there is a piece of hardware with buttons for the 
Icom radios due to its touch screen interface.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by Adriani90** [comment]
@weekscasey how is this now with NVDA 2024.2 Beta 2?

**Comment by Adriani90** [comment]
We didn't get updates from the initial author upon request since more than 6 years, closing as abandoned. if you are still having this issue, please comment and we can reopen.
